### DISCUSSION TITLE
I tried to find the compilerer for this and I could not

### DISCUSSION THREAD

**Discussion by aoirthoir** [TodePond/GulfOfMexico#406] [General]
Is Dreamberd Intapreted or compileded?

**Comment by mybearworld** [comment]
You'll need to install the DreamBerd Installer Installer.

https://github.com/TodePond/DreamBerd/discussions/183

**Comment by aoirthoir** [comment]
That is where i went wrong. I installed the DreamBerd Installer Installer Installer and not the DreamBerd Installer Installer.

Someone really should make this language real tho.
